### ISSUE TITLE
Jump tables located incorrectly

### ISSUE THREAD

**Issue by NandniJamnadas** [pietraferreira/corev-binutils-gdb#9]
Jump table generations are currently stored in the **.text section** rather than the **.rodata section**.

**Comment by pietraferreira** [comment]
To generate the jump table we can run the file below with the following options:

```bash
riscv32-corev-elf-as -march=rv32izcmt testcase.s -o testcase.o
riscv32-corev-elf-ld testcase.o --defsym=callee=0x1000
riscv32-corev-elf-objdump -d a.out
```

Test file:
```assembly
.text                                                                           
    .globl  _start                                                                  
    .type _start, @function                                                         
_start:                                                                             
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee                                                                    
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    call  callee
    .size _start, .-_start
```

**Comment by CharKeaney** [comment]
Fixed in patch #14.
